server/src/main/kotlin/org/javacs/kt/KotlinTextDocumentService.kt

```diff
@@ -65,7 +64,7 @@ class KotlinTextDocumentService(
         get() = uriContentProvider.contentOf(parseURI(uri))
 
     private enum class Recompile {
-        ALWAYS, AFTER_DOT, WAIT_FOR_LINT, NEVER
```

Comment:
As far as I can tell, WAIT_FOR_LINT is not used, and it could interact with the removal of "linting", so removing it as well.